Vagrantfile

```diff
@@ -8,10 +8,10 @@ Vagrant.configure("2") do |config|
   config.vm.box = "puppetlabs/ubuntu-14.04-64-puppet"
   config.vm.box_version = "= 1.0.3"
 
-  config.vm.network "private_network", type: "dhcp"
+  #config.vm.network "private_network", type: "dhcp"
   
   # Use NFS for shared folders for better performance
-  config.vm.synced_folder '.', '/vagrant', nfs: true
+  #config.vm.synced_folder '.', '/vagrant', nfs: true
```

Comment:
you removed the whole shared folders feature, didn't you?
pls check if we can still access the vagrant folder inside the vagrant box

Reply:
No just the optional nfs link. It will be mounted in the box using the builtin virtualbox file mount. This is standard behaviour and doesn't need to be declared.

Reply:
OK